js/recipe/introduction.js

```diff
@@ -0,0 +1,16 @@
+import { htmlElement } from "../htmlRenderer.js"
+import { els } from "../htmlRenderer.js"
+import { render } from "../htmlRenderer.js"
+
+export function introduction(recipe) {
+  const recipeParentElement = document.querySelector("article.post.single");
+  const latterDescriptionParagraphs = recipe.description.slice(1);
+
+  const firstParagraphElement = els.p({innerHTML: recipe.description[0]});
+  const firstParagraphDiv = els.div({className: "first-paragraph", children: [firstParagraphElement]});
+  const introductionParentElement = render(recipeParentElement, els.div({className: "introduction-paragraphs", children: [firstParagraphDiv]}));
+
+  latterDescriptionParagraphs.forEach(paragraph => {
```

Comment:
I wonder if this may rely on the fact that so far we don't clear the targetElement when rendering to it. One requirement we may find very useful about `render` later is that it is idempotent i.e. the html doesn't change if you call it twice with the same things. That'll be useful for re-rendering when the inputs change automatically.
Is there another way to structure this that uses `map` instead, for example? Or something that calls `render` once?

Reply:
I am still working on getting `render` to not repeat the html when it is called twice... but I've managed to update this introduction.js code so that it only calls render once, with all the children of the target node passed in in one go, which is pretty neat!